src/Microsoft.DotNet.Build.Tasks/PackageFiles/scripts/docker/init-docker.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+set -e
+
+scriptName=$0
+
+function usage {
```

Comment:
We've been using `usage() {` syntax, e.g. in https://github.com/dotnet/buildtools/blob/master/bootstrap/bootstrap.sh. We need to figure out what way is supported by the most shells.